### ISSUE TITLE
Can't create category page

### ISSUE THREAD

**Issue by NoahDragon** [hexojs/hexo-theme-landscape#75]
Issue from https://github.com/hexojs/hexo/issues/2170 @phicarre 

> 1) I created a page with the command: hexo new page categories
> 2) In source/categories/index.md, I added type: categories in the front matter
> 3) In the _config.yml of the theme, I added in the menu: Categories: /categories
> but my page is always empty !
> What is missing ?

I also investigate the issue, and couldn't find a solution. Do we support category layout in this theme?

**Comment by itpropro** [comment]
I have the same problem, does this theme support category pages?

**Comment by orcmid** [comment]
Yes, category pages are produced for categories.  Here is my category test confirming operation with my employment of the landscape theme: http://orcmid.com/BlunderDome/wingnut/2017/12/02/Categories/
Notice the category list on the sidebar, and the links to category pages.  (The category page lack a ttie that conveys the category, so that's something to ponder.  It is in the URL.)

To see whatever I did regarding the enabling of category pages, you can see my sources at https://github.com/orcmid/hexo-theme-landscape-wingnut for the landscape theme,

See https://github.com/orcmid/hexo-spanner for category-related information in the overall blog, especially in the _config.yml
